content/blog/posts/cyberforce-2019-story.rst

```diff
@@ -0,0 +1,22 @@
+OSU Security Club Wins Regional Competition
+--------------------------------------------------
+:author: Abby Lawrence
+:date: 2020-02-04
+:img: 2019-Cyberforce-competition-1.jpg
+:slug: cyberforce-2019-story
+:tag: featured-stories
+:tags: student-stories
+
+The Oregon State Security Club has placed first in the regional CyberForce Competition that is hosted yearly 
+by the Department of Energy. This past November is the third consecutive year the OSU computer science students have 
+placed first in the region. The team placed sixth on the national level, which was made up of over 100 teams. 
+
+The members of the team included: Cody Holliday, Ryan Kennedy, Matt Jansen, Khuong Luu, Zach Rogers, and Zander Work. 
+They were advised by assistant professor of computer science Yeongjin Jang.
```

Comment:
This post needs to have some kind of connection to the OSL. I know that Cody and Khoung are/were students of the lab so that would be a good tie in. I recommend you try and get a quote for Cody and/or Khoung and have it relate back to their experience at the OSL as well.